server/routes/approvingLicences/handlers/approvalView.ts

```diff
@@ -49,6 +49,8 @@ export default class ApprovalViewRoutes {
           licence.kind !== LicenceKind.HDC_VARIATION &&
           licence.isDueForEarlyRelease,
         hdcLicenceData,
+        isPPUser: false,
```

Comment:
is this template only ever used from here and for prison users? 
If so don't think we need to pass this and can hardcode it where we need it.